File: C/french/french.c
#include <stdio.h>
#include <string.h>

int main () {

	char *letter;
	int cases, tcount = 0, scount = 0;
	int i = 0;
	scanf("%d", &cases);


	while (i < cases) {
		
		scanf("%c", letter);
		
		if ((strcmp(letter, ".") == 0) || (strcmp(letter, "?") == 0) || (strcmp(letter, "!") == 0)) {
			i += 1;
		} else if (strcmp(letter, "s") == 0 || strcmp(letter, "S") == 0) {
			scount += 1;
		} else if (strcmp(letter, "t") == 0 || strcmp(letter, "T") == 0) {
			tcount += 1;
		}
		
	}

	printf("%d\t%d\n", scount, tcount);

//	if (scount >= tcount) {
//		printf("French\n");
//	} else {
//		printf("English\n");
//	}


	return 0;
}

File: python/french.py
tUpper = 0
tLower = 0
sUpper = 0
sLower = 0
cases = int(input())
for i in range(cases):
    line = str(input())
    tUpper += line.count("T") 
    tLower += line.count("t")
    sUpper += line.count("S") 
    sLower += line.count("s")

tcount = tUpper + tLower
scount = sUpper + sLower
if cases != 0:
    if scount >= tcount:
        print("French")
    elif scount < tcount:
        print("English")